ILSpy/App.xaml

```diff
@@ -25,6 +26,12 @@
 
 		<Style TargetType="ScrollViewer">
 			<Setter Property="toms:AdvancedScrollWheelBehavior.Attach" Value="WithAnimation" />
+			<Style.Triggers>
```

Comment:
As written in the documentation, `Attach` is a shortcut and should only be used if you want to go with the default only.
If you want to control the behavior, you should not attach and detach it, but use the full behavior notation and simply configure the properties of interest. (now I understand the intend of https://github.com/tom-englert/TomsToolbox/pull/20, this was not necessary at all)

Reply:
I was not aware of the method you used in your implementation, hence this rudimentary implementation.

Reply:
Agree this is a bit hidden, in xaml VS does not show the documentation directly when hovering...